vendor/vbom.ml/util/sortorder/natsort.go

```diff
@@ -0,0 +1,76 @@
+package sortorder
+
+// Natural implements sort.Interface to sort strings in natural order. This
+// means that e.g. "abc2" < "abc12".
+//
+// Non-digit sequences and numbers are compared separately. The former are
+// compared bytewise, while the latter are compared numerically (except that
+// the number of leading zeros is used as a tie-breaker, so e.g. "2" < "02")
+//
+// Limitation: only ASCII digits (0-9) are considered.
```

Comment:
I don't believe this is a problem. It looks like the sort works fine with non-ascii characters, but the detection and special handling of digits only considers ASCII digits as opposed to other numerals (https://en.wikipedia.org/wiki/Numerals_in_Unicode).